src/eth/TransactionLifeCycle.js

```diff
@@ -14,6 +15,7 @@ class TransactionLifeCycle {
       transactionTypeTransitions[TransactionType.transaction]
     );
     this._businessObject = businessObject;
+    this._web3Service = new Web3Service();
```

Comment:
Which is preferred in this case, instantiating a new service (like here), or extending the `TransactionLifeCycle` class with the `Web3Service` and using `.get`?

Reply:
Oh, we definitely don't want to instantiate it from scratch. An instance is already being passed into the constructor for TransactionObject, so you just need to add it to the superclass's constructor as well.